engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -1,316 +1,459 @@
+  "binding-behavior-editor": "Přepnout editor chování",
+  "binding-chat": "Přepnout chat",
+  "binding-console": "Přepnout konzoli",
+  "binding-crouch": "Přikrčit / Dolů",
+  "binding-crouch-mode": "Režim přikrčení",
+  "binding-decrease-view-distance": "Snížit viditelnou vzdálenost",
+  "binding-forwards": "Dopředu",
+  "binding-frob": "Použít cílový předmět",
+  "binding-hide-hud": "Skrýt HUD",
+  "binding-increase-view-distance": "Zvýšit viditelnou vzdálenost",
+  "binding-jump": "Skok / Nahoru",
+  "binding-left": "Úkrok vlevo",
+  "binding-nui-editor": "NUI Editor",
+  "binding-nui-skin-editor": "NUI Editor vzhledu",
+  "binding-pause": "Herní menu",
+  "binding-right": "Úkrok vpravo",
+  "binding-screenshot": "Pořídit screenshot",
+  "binding-show-online-players": "Zobrazit online hráče",
+  "binding-toggle-speed-permanently": "Trvale přepnout rychlost",
+  "binding-toggle-speed-temporarily": "Dočasně přepnout rychlost",
+  "binding-toolbar-1": "Toolbar Slot 1",
+  "binding-toolbar-2": "Toolbar Slot 2",
+  "binding-toolbar-3": "Toolbar Slot 3",
+  "binding-toolbar-4": "Toolbar Slot 4",
+  "binding-toolbar-5": "Toolbar Slot 5",
+  "binding-toolbar-6": "Toolbar Slot 6",
+  "binding-toolbar-7": "Toolbar Slot 7",
+  "binding-toolbar-8": "Toolbar Slot 8",
+  "binding-toolbar-9": "Toolbar Slot 9",
+  "binding-toolbar-10": "Toolbar Slot 10",
+  "binding-use-item": "Použít držený předmět",
+  "biome-fog": "Mlha",
+  "biome-temperature": "Teplota",
+  "biome-humidity": "Vlhkost",
+  "biome-name": "Název",
+  "bloom-effect": "Rozostření",
+  "bobbing": "Pohyb hlavy",
+  "button-web-browser-confirmation-title": "Je vyžadováno potvrzení.",
+  "button-web-browser-confirmation-message": "Opravdu chceš otevřít toto url v prohlížeči?",
+  "camera-blur": "Rozmazání",
+  "camera-blur-max": "Maximální",
+  "camera-blur-normal": "Normální",
+  "camera-blur-off": "Vypnuto",
+  "camera-blur-some": "Trochu",
+  "camera-details-settings": "Nastavení kamery / grafického detailu",
+  "camera-film-grain": "Filmové zrnění",
+  "camera-setting": "Nastavení kamery",
+  "camera-setting-cinematic": "Cinematická",
+  "camera-setting-normal": "Normální",
+  "camera-setting-smooth": "Hladká",
+  "category-behavior": "Chování",
+  "category-general": "Obecné",
+  "category-interaction": "Interakce",
+  "category-inventory": "Inventář",
+  "category-mouse": "Myš",
+  "category-movement": "Pohyb",
+  "category-nui": "NUI",
+  "change-keybind-popup-message": "je již přiřazena. Přesto přiřadit?",
+  "change-keybind-popup-title": "Klávesa je už přiřazena",
+  "clamp-lighting": "Zamezit přesvětlení",
+  "cloud-shadows": "Stíny mraků",
+  "config": "Nastavit",
+  "connecting-to": "Připojuji se k",
+  "connection-failed": "Připojení selhalo!",
+  "continue": "Pokračovat",
+  "continue-pregeneration": "Pokračovat k Pregeneraci",
+  "continue-spawn-preview": "Pokračovat k náhledu Spawnu",
+  "continue-universe-setup": "Pokračovat k nastavení Vesmíru",
```

Comment:
```suggestion
  "continue-universe-setup": "Pokračovat k nastavení světa",
```

Reply:
I think this might become misleading. If I get it correctly from the game menus and various texts and hints, we can create a Universe that contains multiple Worlds, although the multi-world is not finished yet and not working.
So I'd go with Universe or something else that can pack multiple worlds inside.